### PR TITLE
[WIP] Feature/new oracle

### PR DESCRIPTION
**Pull request by leonprou**
- new funding with oracle
- adding rewards screen

### CODE REVIEW

File: src/components/PoolDashboard/PoolDashboard.js [hunk 1]
```diff
@@ -33,46 +33,59 @@ const GET_POOL = gql`
...
+      txHash
+      createdAt
+      pool
+      amount
+      receiver
+    }
+  }
+`
+
+const useStyles = makeStyles(theme => ({
```
Comment (leonprou) [inline src/components/PoolDashboard/PoolDashboard.js:51]: sorry, ran my formatter here..

File: src/components/PoolCreator/Wizard/Wizard.js [hunk 2]
```diff
@@ -166,14 +166,20 @@ export const Wizard = (props) => {
 
     // create pool contract on the blockchain
     // Get blockchain data after tx confirms, then update pool object
+
+    const args = [
+      config.network.addresses.cDai,
+      config.network.operator,
```
Comment (tamirse) [inline src/components/PoolCreator/Wizard/Wizard.js:172]: what is operator? can you give it a descriptive name?

File: src/components/PoolCreator/Wizard/Wizard.js [hunk 3]
```diff
@@ -166,14 +166,20 @@ export const Wizard = (props) => {
 
     // create pool contract on the blockchain
     // Get blockchain data after tx confirms, then update pool object
+
+    const args = [
```
Comment (tamirse) [inline src/components/PoolCreator/Wizard/Wizard.js:170]: why did you remove the pool owner address? don't we need it to allow changes to the pool?

File: src/components/PoolCreator/Wizard/Wizard.js (hunk 2 above)
Comment (leonprou) [reply src/components/PoolCreator/Wizard/Wizard.js:172]: operator is our backend account that is responsible to call the oracle in case there was a winner

File: src/components/PoolCreator/Wizard/Wizard.js (hunk 3 above)
Comment (leonprou) [reply src/components/PoolCreator/Wizard/Wizard.js:170]: the second argument was for the node operator actually. we put there the owner just temporary. We know who's the owner cause it's he's the one who sends the tx
